test/integration/setup.js

```diff
@@ -22,16 +20,16 @@ const web3ToEthers = (x, signers) => {
       .map(x => ({ val: new ethers.Contract(contracts[x].address, contracts[x].abi, accounts.defaultSender), key: x }))
       .reduce((acc, x) => ({ ...acc, [x.key]: x.val }), {}),
     rates: { daiToEthRate: parseEther(daiToEthRate.toString()), ethToDaiRate },
-    accounts: accounts,
+    accounts,
   };
 };
 
-async function setup () {
+async function setup() {
   // external
   const ERC20BlacklistableMock = artifacts.require('ERC20BlacklistableMock');
   const OwnedUpgradeabilityProxy = artifacts.require('OwnedUpgradeabilityProxy');
   const ChainlinkAggregatorMock = artifacts.require('ChainlinkAggregatorMock');
-  const Lido = artifacts.require('P1MockLido');
+  // const Lido = artifacts.require('P1MockLido');
```

Comment:
Should this and the lines below be commented out?

Reply:
The varibles were unused but I didn't want to remove the code because it might be needed later (some of the tests are a work in progress). Commented out to keep the code but remove the warning.